src/Blocks/components/menu/styles/vertical/menu-vertical-style.scss

```diff
@@ -21,5 +21,13 @@ $menu-vertical: (
 		padding: map-get-strict($menu-vertical, link-padding);
 
 		@include link($menu-vertical, link-modifiers);
+
+		&:hover {
+			color: $base-black-color;
+		}
+	}
+
+	&__item--highlight &__link {
```

Comment:
this is an odd selector?

this will return `menu-vertical__item--hightlight__link` ??

is this correct?

Reply:
It returns `.menu-vertical__item--highlight .menu-vertical__link`, which is what I need to highlight some menu items per design (in the vertical menu the highlighted item is bold, the rest are semibold)

The `--highlight` class is the one added in the Menu Editor.